aggregator/graphqlResponseGetter.go

```diff
@@ -0,0 +1,42 @@
+package aggregator
+
+import (
+	"context"
+	"encoding/json"
+	"strings"
+
+	"github.com/ElrondNetwork/elrond-sdk-erdgo/authentication"
+	"golang.org/x/oauth2"
+)
+
+// GraphqlResponseGetter wraps over the default http client
+type GraphqlResponseGetter struct {
+	authentication.AuthClient
+}
+
+type graphQLRequest struct {
+	Query     string `json:"query"`
+	Variables string `json:"variables"`
+}
+
+// Query does a get operation on the specified url and tries to cast the response bytes over the response object through
+// the json serializer
+func (getter *GraphqlResponseGetter) Query(ctx context.Context, url string, query string, variables string, response interface{}) error {
```

Comment:
"variables" is a little too generic. What it should contain?

Reply:
this is the name convention for it (more info [here](https://dgraph.io/docs/graphql/api/variables/#:~:text=Variables%20simplify%20GraphQL%20queries%20and,a%20function%20and%20begin%20with%20%24%20.))
I would not change it

Reply:
ok